src/modules/esl-base-popup/README.md

```diff
@@ -8,6 +8,11 @@ Authors: *Julia Murashko*, *Alexey Stsefanovich (ala'n)*.
 
 ESLBasePopup - is a custom element, that is used as a base for "Popup like" components creation.
 
+ESLPopupDispatcher - plugin component, that prevents activation of multiple ESLBasePopup instances in bounds of managed container.
+Usually (and by default) binds to document.body. Use native dom events to manage controlled instances state.
```

Comment:
proposal, may be "DOM events"?

Reply:
Will be updated with the upcoming changes in the separate PR. 
Thanks!